src/main/java/no/sikt/nva/channelregister/ChannelRegister.java

```diff
@@ -87,6 +90,22 @@ private ChannelRegister() {
             getChannelRegisterAliases(UIB_PUBLISHER_CHANNEL_REGISTRY_FACULTIES_CSV_PATH);
         this.channelRegisterAliasesForNmbu = getChannelRegisterAliases(
             NMBU_PUBLISHER_CHANNEL_REGISTRY_FACULTIES_CSV_PATH);
+        this.publisherWildCards = getPublisherWildcards(PUBLISHER_WILDCARDS_CSV_NAME);
+    }
+
+    private List<ChannelRegisterWildcard> getPublisherWildcards(String filename) {
+        try (var inputStream = Thread.currentThread().getContextClassLoader()
```

Comment:
Is there a reason to not use commons here?

Reply:
NVA-commons requires at least java 17, brage machine has java 11.